src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 1.5;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+    color: $additional-text-color;
+  }
+
+  &__reviews-block {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__reviews {
    margin: 0;
    font-size: 10px;
    font-weight: 400;
    line-height: 1.4;
    
    &-block {
      // write styles here
    }
  }
```